applications/StructuralMechanicsApplication/tests/cpp_tests/test_truss_constitutive_law.cpp

```diff
@@ -0,0 +1,59 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "structural_mechanics_fast_suite.h"
+#include "custom_constitutive/truss_constitutive_law.h"
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(TrussConstitutiveLaw_CalculatesLinearElasticStress, KratosStructuralMechanicsFastSuite) {
+    TrussConstitutiveLaw law;
+    ConstitutiveLaw::Parameters parameters;
+    Vector temp_strain = ScalarVector(1, 0.005);
+    Vector temp_stress = ZeroVector(1);
+    parameters.SetStrainVector(temp_strain);
+    parameters.SetStressVector(temp_stress);
+
+    Properties properties;
+    properties[YOUNG_MODULUS] = 1e6;
+    parameters.SetMaterialProperties(properties);
+    law.CalculateMaterialResponsePK2(parameters);
+
+    constexpr double expected_stress = 5000; // = Strain * Young's Modulus
```

Comment:
Also here, I would suggest to use `constexpr` variables to transform the comment into actual code.

Reply:
Done